src/utils/rest-api.js

```diff
@@ -4,82 +4,118 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
+import {store} from '../redux/store'
```

Comment:
Ideally this would not be a global but let's save this for another PR as this is already existing